jmclient/jmclient/wallet_utils.py

```diff
@@ -1222,21 +1222,24 @@ def wallet_gettimelockaddress(wallet, locktime_string):
         jmprint("Error: not a fidelity bond wallet", "error")
         return ""
 
-    m = FidelityBondMixin.FIDELITY_BOND_MIXDEPTH
-    address_type = FidelityBondMixin.BIP32_TIMELOCK_ID
-
     lock_datetime = datetime.strptime(locktime_string, "%Y-%m")
     if jm_single().config.get("BLOCKCHAIN", "network") == "mainnet" and lock_datetime <= datetime.now():
         jmprint("Error: locktime must be a future date", "error")
         return ""
 
+    m = FidelityBondMixin.FIDELITY_BOND_MIXDEPTH
+    address_type = FidelityBondMixin.BIP32_TIMELOCK_ID
```

Comment:
Why did you move these two lines?

Reply:
Because they are not used in check above. Sould have done this way already in #973.

Reply:
I see; but does it make any functional difference?

I realise it doesn't matter. I'm asking only because I'm afraid I've misunderstood something.

Reply:
Totally no difference. Just moved the same lines little bit below. And tested that getting new timelock address works after that change,